src/lib/hooks/routing/useClientSideSmartOrderRouterTrade.ts

```diff
@@ -10,7 +11,16 @@ import { getClientSideQuote } from './clientSideSmartOrderRouter'
 import { useRoutingAPIArguments } from './useRoutingAPIArguments'
 
 const protocols: Protocol[] = [Protocol.V2, Protocol.V3]
-const config = { protocols }
+
+// The following configs are used to reduce latency when using the client side SOR
+const MIN_DISTRIBUTION_PERCENT = 5
+
+const config = {
+  protocols,
+  distributionPercent: MIN_DISTRIBUTION_PERCENT,
+  v3PoolProvider: StaticV3SubgraphProvider,
```

Comment:
This worries me a little bit:

https://github.com/Uniswap/smart-order-router/blob/b19ebcb3f3e2b6b10a8021884f5336c8735ba8a5/src/providers/v2/static-subgraph-provider.ts#L49-L50

Reply:
I also learned that teh hoardcoded provider only helps on the first load, so probably not worth the change here